src/boot/efi/linux.c

```diff
@@ -51,12 +56,11 @@ EFI_STATUS linux_exec(EFI_HANDLE image,
                 return EFI_LOAD_ERROR;
 
         addr = UINT32_MAX; /* Below the 32bit boundary */
-        err = uefi_call_wrapper(
-                        BS->AllocatePages, 4,
-                        AllocateMaxAddress,
-                        EfiLoaderData,
-                        EFI_SIZE_TO_PAGES(0x4000),
-                        &addr);
+        err = uefi_call_wrapper(BS->AllocatePages, 4,
+                                AllocateMaxAddress,
+                                EfiLoaderData,
+                                EFI_SIZE_TO_PAGES(0x4000),
+                                &addr);
```

Comment:
why this change? usually our coding style breaks before the first argument

Reply:
Coding style only mentions function declarations. (Indent parameter list by 16 spaces); This change was made so that calls in linux.c have the same indention style also used in some code style (first parameters on the same line) for call_wrapper is used in boot.c and stub.c

See also https://github.com/systemd/systemd/pull/20918#discussion_r720708817. (or have I misunderstood @medhefgo  there?)

Reply:
back then clang simply didn't support our formatting (iirc it insisted on putting the opening { of functions on a separate line). Also it fucked up all aligned structure array tables we have, i.e. removed the whitespace that aligned the columns. And we just had so many of them that disabling the formatting logic for each wasjust too much.